### ISSUE TITLE
Failed to import `miragejs`

### ISSUE THREAD

**Issue by bertdeblock** [adopted-ember-addons/ember-file-upload#683]
`5.0.0-beta.5` breaks our build with the following error:

```
Build Error (WebpackBundler)

ember-file-upload tried to import "miragejs" in "ember-file-upload/mirage/index.js" but the package was not resolvable from /Users/bertdeblock/Code/bagaar/projects/mobility-buddy/mobility-buddy-frontend/node_modules/ember-file-upload


Stack Trace and Error Report: /var/folders/q5/sg7x51yn0v5d4snmcrmcxxk00000gn/T/error.dump.ea295ef192933e99591559f0aa01612f.log
```

Installing `miragejs` ourselves fixes the issue, but our project doesn't use / need MirageJS.

**Comment by gilest** [comment]
This will be related to #664

Are you using embroider? I'll try to investigate and repro tomorrow

**Comment by bertdeblock** [comment]
We are not at the moment. Thanks! Let me know if you need any additional info.

**Comment by pjcarly** [comment]
I am trying out embroider, and seeing this issue (with beta-5), not with (beta-3)

**Comment by gilest** [comment]
I have a fix in #684 which I have tested manually in all four scenarios eg. with/without embroider and with/without mirage.

I'm not merging right away as the embroider CI builds are unhappy.

It would be helpful to me in the meantime if you could please try with the fix installed and report back:
```
"ember-file-upload": "adopted-ember-addons/ember-file-upload#fix/auto-import-exlcude-miragejs",
```

**Comment by pjcarly** [comment]
I can confirm my build is succeeding with your fix. Cheers

**Comment by gilest** [comment]
Fixed in `v5.0.0-beta.6`
